### ISSUE TITLE
Keeps on creating a blank terminal with no prompt.

### ISSUE THREAD

**Issue by bassamanator** [jeremyramin/terminal-plus#582]
![image](https://user-images.githubusercontent.com/61985779/84813027-5102e980-afdd-11ea-9594-d2b8da9b9c8b.png)

That's what I get when I hit the +. Creates a blank terminal. Atom is up to date so are my packages apparently.
:/

**Comment by sgodau** [comment]
Yes, I have the same error. I am new to Atom and just installed it on a Windows 10 and on a Ubuntu system. On both systems I get a blank terminal.

**Comment by ayushdiagarwal** [comment]
Same error on Ubuntu 20.04.

**Comment by ozeecode** [comment]
Same issue for me on Windows 10 I also tried different terminal packages for Atom. They have exactly the same problem. Maybe this can be related to Atom?

**Comment by granolocks** [comment]
:+1: 
![image](https://user-images.githubusercontent.com/209583/86501549-c99cd080-bd67-11ea-99e0-766e86ad1504.png)
```
user@system:~$ uname -a
Linux system 5.3.0-55-generic #49-Ubuntu SMP Thu May 21 12:47:19 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
user@system:~$ atom -v
Atom    : 1.41.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0

```

**Comment by Duser919** [comment]
Same issue. Similar issue in platform-ide-terminal too

Did you guys have any installation errors?

**Comment by bassamanator** [comment]
@Duser919 Nope. Just install termination, it's quite nice.

**Comment by KontrivedMedia** [comment]
I get the same issue. Has anyone fixed this yet?

19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64 x86_64
Atom    : 1.48.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0
gcc version 10.1.0 (Homebrew GCC 10.1.0) 
node-gyp -v v7.0.0

**Comment by lordcenzin** [comment]
Same issue here on Arch Linux with last version of Atom

```
$ uname -a
Linux kosung 5.7.10-arch1-1 #1 SMP PREEMPT Wed, 22 Jul 2020 19:57:42 +0000 x86_64 GNU/Linux
$ atom -v
Atom    : 1.49.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0

```

**Comment by narayan-iyer-hello** [comment]
Same issue here on macOS:

```
➜  uname -a
Darwin macbook-pro.lan 19.6.0 Darwin Kernel Version 19.6.0: Sun Jul  5 00:43:10 PDT 2020; root:xnu-6153.141.1~9/RELEASE_X86_64 x86_64
```

```
➜  atom -v
Atom    : 1.49.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0
```

**Comment by adamrpostjr** [comment]
Same issue here
`Linux CommandCenter 5.7.12-arch1-1 #1 SMP PREEMPT Fri, 31 Jul 2020 17:38:22 +0000 x86_64 GNU/Linux`
`Atom    : 1.49.0
Electron: 5.0.13
Chrome  : 73.0.3683.121
Node    : 12.0.0`

**Comment by straightcurve** [comment]
same here
`Linux ws-remote 5.8.3-zen1-1-zen #1 ZEN SMP PREEMPT Fri, 21 Aug 2020 16:54:22 +0000 x86_64 GNU/Linux`

**Comment by straightcurve** [comment]
just saw this #577

**Comment by straightcurve** [comment]
the one from #577 doesnt work either, i got it working with this one, a fork from the one from #577 https://github.com/bus-stop/terminus

**Comment by ScamCast** [comment]
Same issue here on Windows 10

@straightcurve Thanks for that fork. Installed terminus and it's now working :)

**Comment by jadeye** [comment]
Same here on ubuntu 18.04
